### PR TITLE
blaze: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/blaze/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{% set name = "blaze" %}
+{% set version = "0.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/blaze/blaze.git
+  gti_tag: {{ version }}
```
Comment (ocefpaf) [inline recipes/blaze/meta.yaml]: I think `gti` is a typo, right? Can we use the tarball format instead? It saves us from cloning the repo.

Comment (nehaljwani) [inline recipes/blaze/meta.yaml]: I fixed the typo. The tarball isn't available as they didn't cut a release for that tag and they also don't have a [release on pypi](https://github.com/blaze/blaze/issues/1654). :'(

Comment (ocefpaf) [inline recipes/blaze/meta.yaml]: You can still use a "tarball" with

```yaml
  url: https://github.com/blaze/blaze/archive/{{ version }}.tar.gz
  sha256: <sha256sum of https://github.com/blaze/blaze/archive/0.11.3.tar.gz>
```

**Comment by notestaff** [comment]
blaze does exist in the 'defaults' channel...

File: recipes/blaze/meta.yaml (hunk 1 above)
Comment (nehaljwani) [reply recipes/blaze/meta.yaml]: Ah, I didn't know that it worked for tags. Cool! I have updated the PR.

**Pull request merged**